projects/configs/tag-operations/mri.yaml

```diff
@@ -126,6 +98,9 @@
   group: 0x0010
   element: 0x0040
   op: "keep"
```

Comment:
patient sex should only be in base

Reply:
[0b63011](https://github.com/SAFEHR-data/PIXL/pull/498/commits/0b63011e0e8e6bce28b7e64ccba7f9e733bff945)